src/Symfony/Component/Cache/Tests/Adapter/MemcachedAdapterTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Adapter;
+
+use Symfony\Component\Cache\Adapter\MemcachedAdapter;
+
+/**
+ * @author Rob Frawley 2nd <rmf@src.run>
+ */
+class MemcachedAdapterTest extends AdapterTestCase
+{
+    protected $skippedTests = array(
+        //'testExpiration' => 'Testing expiration slows down the test suite',
+        //'testHasItemReturnsFalseWhenDeferredItemIsExpired' => 'Testing expiration slows down the test suite',
+        //'testDefaultLifeTime' => 'Testing expiration slows down the test suite',
+    );
+
+    /**
+     * @var \Memcached
+     */
+    protected static $client;
+
+    protected static function defaultClientServer()
+    {
+        return sprintf(
+            'memcached://%s:%d',
+            getenv('MEMCACHED_HOST') ?: '127.0.0.1',
+            getenv('MEMCACHED_PORT') ?: 11211
+        );
+    }
+
+    private static function defaultClientOptions()
+    {
+        return array(
+            'OPT_LIBKETAMA_COMPATIBLE' => true,
+            'OPT_COMPRESSION' => true,
+        );
+    }
+
+    public static function setupBeforeClass()
+    {
+        if (!MemcachedAdapter::isSupported()) {
+            self::markTestSkipped('Extension memcached >2.1.0 required.');
```

Comment:
`>2.1.0` and `>=2.2.0` are the same thing, but I follow the logic of clarifying this. :+1: